.circleci/config.yml

```diff
@@ -1,29 +1,56 @@
 version: 2
 jobs:
+  integration-php72-magento23:
+    docker:
+      - image: boltdev/m2-plugin-ci-integration-php72:v0
+        auth: 
+          username: ethanwayda22  # can specify string literal values
```

Comment:
i feel like we can definitely pass in env variables here, lets look into this when pairing tomorrow